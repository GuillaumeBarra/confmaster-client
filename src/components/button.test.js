import React from 'react'
import {render} from '@testing-library/react'
import Button from './button'
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<Button />)
    
    expect(asFragment(<Button />)).toMatchSnapshot()
   })