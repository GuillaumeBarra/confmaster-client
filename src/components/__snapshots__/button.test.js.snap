// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <button
      style="height: 4vh; width: 4vh; border-radius: 50%; font-size: 22px; color: white; background-color: rgb(126, 67, 215); border-width: 0px; cursor: pointer;"
    >
      +
    </button>
  </div>
</DocumentFragment>
`;
